.github/workflows/run_publish_package.yaml

```diff
@@ -0,0 +1,56 @@
+on: 
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    paths:
+      - ".github/workflows/run_publish_package.yaml"
+      - "utils/DeployUtils.ps1"
+      - "Testing/Functional/BuildTest/ModuleBuild.Tests.ps1"
+
+name: Publish Package
```

Comment:
Consider adding a comment or two that explains the purpose of this workflow and gives any important details.